### PR TITLE
New version: Kopia.KopiaUI version 0.16.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146211)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Review by OfficialEsco — APPROVED** [review]

File: manifests/k/Kopia/KopiaUI/0.16.0/Kopia.KopiaUI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-03-23
+AppsAndFeaturesEntries:
+- DisplayName: KopiaUI 0.14.1
```
Comment (OfficialEsco) [inline manifests/k/Kopia/KopiaUI/0.16.0/Kopia.KopiaUI.installer.yaml:16]:
```suggestion
- DisplayName: KopiaUI 0.16.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
shit missed this one

**Comment by stephengillie** [comment]
Close with reason: Duplicate of #146256;

**Comment by stephengillie** [comment]
Closing as duplicate because the other PR has the updated `DisplayName`. But this manifest has much more `ReleaseNotes` data, which should probably still be added into the repo.

**Pull request closed without merging**